src/main/java/core/basesyntax/Splitter.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public class Splitter {
+    public ReportItem splitString(String string) {
+        String[] stringArray = string.split(",");
+        Name name = Name.BUY.name().equalsIgnoreCase(stringArray[0]) ? Name.BUY : Name.SUPPLY;
+        return new ReportItem(name, Integer.parseInt(stringArray[1]));
```

Comment:
The method 'splitString' does one task and is within the recommended line count. However, consider handling potential exceptions that may arise from parsing integers or from array access if the input string does not contain a comma or has insufficient parts after splitting.